designing-event-driven-processes/README.md

```diff
@@ -0,0 +1,18 @@
+# Beyond basic messaging: Designing long-running processes
+
+## Abstract
+
+Messaging in distributed systems offers a range of valuable benefits: it enables the design of decoupled components, where each component can evolve and scale individually. In addition, messaging improves the system’s fault tolerance but goes even further by opening the door to building self-healing systems. One benefit that doesn’t usually appear in the Googleable list of benefits is one of the most powerful: supporting real-life, long-running business processes and making the system mirror real-world business use cases. And that’s exactly the focus of this workshop.
```

Comment:
> but goes even further by opening the door to building self-healing systems

I'm not clear what this means exactly

Reply:
My thought process was that introducing queues in the middle automatically allows for better fault tolerance, in the sense that you never lose requests. By "goes further by opening the door to self-healing systems", I mean that this can be further leveraged to build self-healing systems by implementing retries, for example, as a self-healing mechanism. 

With that insight, do you have any ideas on why that was unclear from the framing? Do you think I'm making too many assumptions?